library/aes.c

```diff
@@ -880,7 +880,7 @@ int mbedtls_aes_crypt_cbc( mbedtls_aes_context *ctx,
                     const unsigned char *input,
                     unsigned char *output )
 {
-    int i;
+    int i, ret;
```

Comment:
`ret` should be initialized as the body of the function doesn't alter it for `length == 0`.